socorro/tests/external/legacy_es/__init__.py

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from distutils.version import LooseVersion
+from functools import wraps
+
+import pytest
+
+
+# Uncomment these lines to decrease verbosity of the elasticsearch library
+# while running unit tests.
+# import logging
+# logging.getLogger('elasticsearch').setLevel(logging.ERROR)
```

Comment:
they haven't been modified to log differently, so I assume they do

Reply:
In elasticsearch-py 1.9.0, they're explicitly creating an "elasticsearch" logger rather than creating a logger using "__name__".

https://github.com/elastic/elasticsearch-py/blob/91cd918477ab0cfcc3a7236b7f651d8254045748/elasticsearch/connection_pool.py#L12

https://github.com/elastic/elasticsearch-py/blob/91cd918477ab0cfcc3a7236b7f651d8254045748/elasticsearch/connection/base.py#L9

Looks like they're still doing that:

https://github.com/elastic/elasticsearch-py/blob/14e6265bf7240d87a5907eadd5fe6d2f8c861265/elasticsearch/__init__.py#L42

Ergo, both the shadowed and current ES libraries are using the "elasticsearch" logger.